src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder results = new StringBuilder();
+        String importantSymbol = ",";
+        int operationTypeIndex = 0;
+        int amountIndex = 1;
+        int arraysTheoryMaxSize = 10;
+        int realSize = 0;
+        String[] type = new String[arraysTheoryMaxSize];
+        int[] amount = new int[arraysTheoryMaxSize];
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String value = reader.readLine();
+            while (value != null) {
+                String[] values = value.split(importantSymbol);
+                for (int i = 0; i < arraysTheoryMaxSize; i++) {
+                    if (type[i] == null && values[operationTypeIndex] != null) {
+                        type[i] = values[operationTypeIndex];
+                        amount[i] += Integer.parseInt(values[amountIndex]);
+                        realSize = i + 1;
+                        break;
+                    }
+                    if (type[i] != null && type[i].equals(values[operationTypeIndex])) {
+                        amount[i] += Integer.parseInt(values[amountIndex]);
+                        break;
+                    }
+                }
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("canot read from file", e);
```

Comment:
The exception message contains a typo. It should be 'cannot' instead of 'canot'. Also, it's better to include the file name in the exception message for more clarity.